birdhouse/config/canarie-api/.gitignore

```diff
@@ -1,2 +1,5 @@
 docker_configuration.py
-conf.extra-service.d/canarie-api.conf
```

Comment:
@mishaschwartz Our staging env found this file became "new" file for git and broke the subsequent autodeploy !

Could you re-add this file to .gitignore?

Reply:
@tlvu I've added a PR #314 . Is this what is breaking the Jenkins build as well do you think or is that unrelated?

Reply:
Completely un-related to Jenkins.  I merged https://github.com/bird-house/birdhouse-deploy/pull/308 yesterday and today I went onto our staging machine and see it still did not autodeploy that merge.  So I checked the logs and found out the reason.